sos/report/plugins/filesys.py

```diff
@@ -75,18 +72,36 @@ def setup(self):
         if self.get_option('lsof'):
             self.add_cmd_output("lsof -b +M -n -l -P", root_symlink="lsof",
                                 priority=50)
+        option_list = [
+            PluginOpt('dumpe2fs', default=False, desc='dump filesystem info'),
+            PluginOpt('frag', default=False,
+                      desc='collect filesystem fragmentation status')
+        ]
 
+    def setup(self):
```

Comment:
Oh, missed this on my first review. This seems like a copy/paste error. The options_list shouldn't go within `setup()`, and we've got a duplicate definition of the `setup()` method, which would break several things. If I'm reading everything right on the github webui, I think you just need to remove these lines entirely.

Reply:
Yeah copy&paste error